src/plugins/lwhmediaplay/README.md

```diff
@@ -0,0 +1,61 @@
+Media Play Plugin
+=================
+
+The mediaplay plugin takes care of various embedded video or audio elements.
+
+First of all, the "autostart" attribute allows a video or audio to start 
+automatically when the step has been entered.
+
+Example:
+	<!-- start this video when the step becomes active	-->
+	<video width="800" height="600" autostart>
+		...
+	</video>
+ 	
+Do not confuse this with the official "autoplay" attribute: This will play the video 
+as soon as the presentation is opened.
+When the step is left, the video (or audio) will be paused (not stopped), so it will
+continue when the step is re-entered, e.g. by stepping back.
+
+In the console, the video will play as well, but without sound to avoid echoes.
+This can be avoided by using the class "noconsole", which will either show an 
+alternative image given in the video tag or, by default, the image 
+"./media/Video-default.png", which must be provided by the user. 
+If neither exist, the video file name will be displayed in a box.
```

Comment:
At least not always. Sometimes You might want to comment on certain frames in the video, so instead of turning away from the audience You better watch it in the console.

Reply:
Well, in any case the video should not play in the preview window, that is showing the *next* slide.

For the current slide, the challenge is that we cannot guarantee that the video for the audience and the speaker are in sync. I guess that could be enforced with additional javascript, but doesn't happen by itself.